galaxy.yml

```diff
@@ -14,3 +14,11 @@ documentation: https://docs.ansible.com/ansible/latest/collections/community/gen
 homepage: https://github.com/ansible-collections/community.general
 issues: https://github.com/ansible-collections/community.general/issues
 #type: flatmap
+build_ignore:
+  - .azure-pipelines
+  - .github
+  - .gitignore
+  - changelogs
```

Comment:
```suggestion
```
It is absolutely essential that changelogs/changelog.yaml is included in the build. Otherwise building the Ansible changelog will not work.

Reply:
Can you expand on that ? I was under the impression that leaving CHANGELOG.rst was sufficient ?